doc/modules/feature_selection.rst

```diff
@@ -267,9 +283,6 @@ of features non zero.
      Journal of the Royal Statistical Society, 72 (2010)
      http://arxiv.org/pdf/0809.2932
 
-   * F. Bach, "Model-Consistent Sparse Estimation through the Bootstrap"
-     http://hal.inria.fr/hal-00354771/
```

Comment:
I think that Bach's procedure (the Bolasso) does things somewhat different from what is implemented in `sklearn`. Also, what is implemented does match Meinshausen & Buhlmann's Stability selection procedure.

Specifically, Bach's Bolasso gets bootstrap replicates of the data, via sampling data pairs with replacement or residuals of a lasso fit, then fits a regular lasso to each bootstrap replicate data set, and at last, intersects the active sets across replicates. It also gives theoretical guidance as to how to choose the regularization parameter for the lasso in both high and low dimensional setting.

On the other hand, Meinshausen & Buhlmann's stability procedure get several subsamples of the data pairs (this is a sample of a smaller size than the original and sampled without replacement), then fits the modified lasso where the penalty of a random subset of features is scaled, and then counts the number of times each feature is selected across subsamples for each value of the regularization parameter. This gives a score for each feature and regularization parameter, which is then thresholded. Then the paper tell you to pick for each feature the highest score across the thresholded regularization parameter.

So, I think they are different. What do you think? I might have missed something.